packages/components/src/internal/components/lineage/node/LineageDetail.tsx

```diff
@@ -1,22 +1,88 @@
+
+const ADDITIONAL_DETAIL_FIELDS = ['properties'];
 
 export interface LineageDetailProps {
     item: Experiment.LineageItemBase;
 }
 
+const LineageDetailImpl: FC<LineageDetailProps & InjectedQueryModels> = memo(props => {
+    const { actions, queryModels } = props;
+    if (queryModels.model.isLoading) return <LoadingSpinner />;
+
+    const additionalCols = queryModels.model.allColumns.filter(
+        col => ADDITIONAL_DETAIL_FIELDS.indexOf(col.fieldKey?.toLowerCase()) > -1
+    );
+    const detailColumns = [...queryModels.model.detailColumns, ...additionalCols];
+
+    return (
+        <DetailPanel
+            model={queryModels.model}
+            actions={actions}
+            queryColumns={detailColumns}
+            detailRenderer={_resolveDetailRenderer}
+        />
+    );
+});
+
+const LineageDetailWithQueryModels = withQueryModels<LineageDetailProps>(LineageDetailImpl);
+
 export const LineageDetail: FC<LineageDetailProps> = memo(({ item }) => {
-    const queryConfig: QueryConfig = useMemo(
+    const queryConfigs = useMemo<QueryConfigMap>(
         () => ({
-            baseFilters: item.pkFilters.map(pkFilter => Filter.create(pkFilter.fieldKey, pkFilter.value)),
-            containerPath: item.container,
-            schemaQuery: SchemaQuery.create(item.schemaName, item.queryName),
+            model: {
+                baseFilters: item.pkFilters.map(pkFilter => Filter.create(pkFilter.fieldKey, pkFilter.value)),
+                containerPath: item.container,
+                schemaQuery: SchemaQuery.create(item.schemaName, item.queryName),
+                requiredColumns: ['*'],
+            },
         }),
         [item]
     );
 
-    // Without providing "key" the DetailPanelWithModel will stop updates
-    return <DetailPanelWithModel key={item.lsid} queryConfig={queryConfig} />;
+    // providing "key" to allow for reload on lsid change
+    return <LineageDetailWithQueryModels key={item.lsid} autoLoad queryConfigs={queryConfigs} item={item} />;
+});
+
+interface RendererProps {
+    data: Map<string, any>;
+}
+
+// exported for jest testing
+export const CustomPropertiesRenderer: FC<RendererProps> = memo(props => {
+    const { data } = props;
+
+    return (
+        <table className="lineage-detail-prop-table">
+            <tbody>
+                {data.map(row => {
+                    const fieldKey = row.get('fieldKey');
+                    const name = fieldKey.substring(fieldKey.indexOf('#') + 1);
+
+                    return (
+                        <tr key={fieldKey}>
+                            <td className="lineage-detail-prop-cell">{name}</td>
+                            <td className="lineage-detail-prop-cell">{row.get('value')}</td>
+                        </tr>
+                    );
+                })}
+            </tbody>
+        </table>
+    );
 });
+
+function _resolveDetailRenderer(column: QueryColumn) {
+    let renderer = resolveDetailRenderer(column);
```

Comment:
Hmm, makes me think we should consider falling back to `resolveDetailRenderer` and then forcing users to opt out of default rendering.